pkg/kubelet/cm/container_manager_linux_test.go

```diff
@@ -63,6 +64,9 @@ func TestCgroupMountValidationSuccess(t *testing.T) {
 }
 
 func TestCgroupMountValidationMemoryMissing(t *testing.T) {
+	if cgroups.IsCgroup2UnifiedMode() {
+		return
```

Comment:
nit: Using t.Skip makes it easier to debug tests since they are skipped instead of silently passing.
 
```suggestion
		t.Skip("skipping cgroup test because [...]")
```